aml_interface/aml_interface.py

```diff
@@ -0,0 +1,217 @@
+import logging
+import os
+import shutil
+
+from azure.ai.ml import MLClient
+from azure.ai.ml.entities import BuildContext, Environment, ManagedIdentityConfiguration
+from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential
+
+logger = logging.getLogger(__name__)
+
+
+def delete_file(file_path):
+    try:
```

Comment:
I saw this method in the other CVToolkit too!

Reply:
Yes, I had to duplicate it because I didn't want the AMLInterface to depend on our toolkit.
Mainly because this package will not be used only by us and it doesn't make sense for those who want to use it to also have to install our toolkit.